docs/content/en/docs/configuration/keys.md

```diff
@@ -1979,6 +1979,7 @@ See also:
 | `modsecurity-timeout-idle`       | `Global` | `30s`   |       |
 | `modsecurity-timeout-processing` | `Global` | `1s`    |       |
 | `modsecurity-timeout-server`     | `Global` | `5s`    | v0.10 |
+| `modsecurity-args`               | `Global` | `unique-id method path query req.ver req.hdrs_bin req.body_size req.body` | v0.14 |
```

Comment:
Use alphabetical order here. Add also in the main table, about the line 410. The format you chose is fine, we don't want to align the whole table with this new width.